src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java

```diff
@@ -1948,7 +1948,7 @@ private static LambdaForm makeLoopForm(MethodType basicType, BasicType[] localVa
         }
         @Override
         public String toString() {
-            StringBuffer sb = new StringBuffer("LoopClauses -- ");
+            StringBuilder sb = new StringBuilder("LoopClauses -- ");
             for (int i = 0; i < 4; ++i) {
                 if (i > 0) {
                     sb.append("       ");
```

Comment:
Consider using `String.repeat` here and on L1960 for clarity.

Reply:
I'm not sure how String.repeat can be used here. Repeated String is not constant and different for each iteration.

Reply:
Long runs of whitespace, especially in blank strings, may have poor readability. I was thinking that

```
" ".repeat(7) and " ".repeat(10)
``` 

might read better than 

```
"       " and "          "
```

respectively. If you don't agree, then simply disregard what I said. If you agree but concerned about negative performance implications, consider creating these strings before the respective for-loops.

Reply:
I suggest we ignore the use case for `String.repeat` here, and leave the patch as is.